src/rviz/ogre_helpers/render_widget.cpp

```diff
@@ -81,12 +81,12 @@ RenderWidget::RenderWidget( RenderSystem* render_system, QWidget *parent )
 
 #if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)
   QApplication::syncX();
-  double pixel_ratio = 1.0;
```

Comment:
This variable should stay double, shouldn't it?

Reply:
Sorry for the late reply - I haven't checked on this PR in almost a year... Yes it should be a `double`. Not sure why I made it an `int` at the time :thinking: .

Reply:
Just recompiled with `double` and realized why...

```
 error: invalid operands of types ‘double’ and ‘int’ to binary ‘operator%’
     render_window_->resize( width()*pixel_ratio_ + (width()*pixel_ratio_ % 2), height()*pixel_ratio_ );
```
It's easily fixed with the following:
```
render_window_->resize( width()*pixel_ratio_ + (static_cast<int>(width()*pixel_ratio_) % 2), height()*pixel_ratio_ );
```
But I haven't tested yet on my mac to see if everything looks good. I will try to do this later in the week.

Reply:
Checked the new changes on my mac and everything seems to be working properly.